### PR TITLE
docs(readme): update readme with links to new packages

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Readme is missing packages.

## What is the new behavior?
Readme has new packages.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,39 +17,38 @@ Daffodil is a frontend toolkit that intends to accomplish three things:
 
 * :free: Forever Free, Open Source, and MIT Licensed
 * :hammer: Platform Agnostic Drivers For Your Platform of Choice
-    * Shopify
-    * Magento 2
-    * In-Memory Backend
-        * Prebuilt
+    * Shopify (WIP, 5% coverage)
+    * Magento 2 (WIP, 50% coverage)
+    * In-Memory Backend (60% coverage)
```
Comment (griest024) [inline README.md]: What does "coverage" mean here?

File: README.md [hunk 2]
```diff
@@ -17,39 +17,38 @@ Daffodil is a frontend toolkit that intends to accomplish three things:
...
-    * Checkout
-* :books: Well Documented
-    * Example Implementation
-    * API References
-    * Tutorials
+* :cake: [Decoupled and Composable Packages for your use case](#packages)
+* :books: Well Documented ([WIP: We need **your** help!](https://github.com/graycoreio/daffodil/issues/336))
 * :bulb: Component Kit
-    * Purely Functional
+    * Functionally Written (Lambda-like)
```
Comment (xelaint) [inline README.md]: What does `Lambda-like` mean?

File: README.md (hunk 1 above)
Comment (damienwebdev) [reply README.md]: I intended coverage to mean the 5% complete of a "stock" ecommerce site. I agree that this is vague.

File: README.md (hunk 2 above)
Comment (damienwebdev) [reply README.md]: I'll add a doc describing this in another PR and revert this change.

File: README.md (hunk 1 above)
Comment (griest024) [reply README.md]: Maybe "implemented" would be a better word.

Comment (damienwebdev) [reply README.md]: I've updated this.

File: README.md (hunk 2 above)
Comment (damienwebdev) [reply README.md]: I've updated this.

**Pull request merged**